metron-platform/metron-management/README.md

```diff
@@ -263,6 +403,105 @@ The functions are split roughly into a few sections:
     * aggregatorConfig - Optional config for aggregator
   * Returns: The String representation of the enrichment config
 
+#### Example
```

Comment:
If we move to use `####` for functions instead of bullets, this should move to `#####`.